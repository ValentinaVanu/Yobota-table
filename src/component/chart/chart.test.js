import {YobotaChart} from '.'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'

test('renders yobota table body component', () => {
  const store = {
    getState: () => ({
      yobota: {
        data: []
      }
    }),
    subscribe: jest.fn(),
    dispatch: jest.fn(),
  }
  const component = renderer.create(
    <Provider store={store}>
      <YobotaChart />
    </Provider>
  ).toJSON()
  expect(component).toMatchSnapshot()
})
