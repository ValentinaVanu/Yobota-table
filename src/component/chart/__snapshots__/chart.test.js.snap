// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders yobota table body component 1`] = `
<div
  className="sc-bdfBwQ Dptgt"
/>
`;
